downstream/modules/automation-hub/proc-configure-ansible-galaxy-cli-verify.adoc

```diff
@@ -0,0 +1,50 @@
+[id="proc-configure-ansible-galaxy-cli-verify_{context}"]
+
+= Configuring Ansible-Galaxy CLI to verify collections
+
+You can configure Ansible-Galaxy CLI to verify collections. This assures that collections you download are approved by your organization and have not been changed after it was uploaded to  {HubName}.
+
+.Prerequisites
+
+* Signed and certified collections have been published to {HubName}.
+* Public key for verification has been added to the local system keyring.
+
+.Procedure
+
+If a collection has been signed by the Galaxy server, the server provides ASCII armored, detached signatures to verify the authenticity of the MANIFEST.json before using it to verify the collection’s contents. You must opt into signature verification by link:https://docs.ansible.com/ansible/devel/reference_appendices/config.html#galaxy-gpg-keyring[configuring a keyring] for `ansible-galaxy` or providing the path with the `--keyring` option.
+
+. To import a public key into a keyring for use with `ansible-galaxy`, run the following command.
```

Comment:
into a keyring "which is not default"

Reply:
@anshulbehl - I was not clear on what you meant here. Did you mean the following: "To import a public key into a unique keyring to use with `ansible-galaxy` CLI..."?

When I read your comment at first, I tested things literally and here is what it seemed like you were saying: "To import a public key into a keyring, which is not default, for use with `ansible-galaxy`..." This option seemed a bit awkward in meaning. Can you provide a little more clarification here so that I'm clearly saying what should be said here? Thanks, @anshulbehl.